### ISSUE TITLE
Update qm manpage to fix install instructions

### ISSUE THREAD

**Issue by aesteve-rh** [containers/qm#671]
Currently installing as instructed in the man at: https://github.com/containers/qm/blob/main/qm.8.md?plain=1#L68
Results in an error:
```
# dnf install --installroot=/usr/lib/qm/rootfs setools-console
error: cannot open Packages database in /usr/lib/qm/rootfs/var/lib/rpm
Error: Error: rpmdb open failed
```
Probably since `/usr/lib/qm/rootfs` is just partial rootfs, and `/etc/qm` and `/var/qm` may be needed as well.

Explore the way to handle it and document it properly.

**Comment by Yarboa** [comment]
@aesteve-rh 
Probably this one will do the trick
```
 dnf install --setopt=reposdir=/etc/qm/yum.repos.d --installroot=/usr/lib/qm/rootfs -y <pkg>
```

Did you give it a try?

**Comment by aesteve-rh** [comment]
> Did you give it a try?

Yeah, it does not work for me either.

**Comment by mkemel** [comment]
The reason it stopped working is that we moved `/var/lib/rpm` to `/usr/share/rpm`, and put an absolute symlink `/var/lib/rpm -> /usr/share/rpm`. Now it was changed to a relative one https://gitlab.com/CentOS/automotive/src/automotive-image-builder/-/merge_requests/181.

When the symlink is fixed (you can fix it in place as well with `ln -sf ../../usr/share/rpm /var/qm/lib/rpm`) - the above command will work. BUT, `/var` and `/etc` in qm rootfs are not the correct ones, and we've been doing it wrong until now. Alex has suggested a script that would do it the right way: https://gitlab.com/CentOS/automotive/src/automotive-image-builder/-/merge_requests/181#note_2294016285

**Comment by nsednev** [comment]
Created draft [MR](https://github.com/containers/qm/pull/788).

**Comment by nsednev** [comment]
@aesteve-rh  I don't think that we have to change anything in the documentation as packages being installed flawlessly.

**Comment by nsednev** [comment]
Closing this ticket as from the perspective of the customers qm should not be manually changed with its packages and as the current documentation works as expected with no need of modifying it.
